### DISCUSSION TITLE
Recent Yi ipcam Dome Guard 1080 serial RFUS... Are new the Yi cams still compatible?

### DISCUSSION THREAD

**Discussion by CP52** [roleoroleo/yi-hack-Allwinner-v2#565] [General]
Since the Yi Cameras descripted in several  examples (YT or HA blogs from 2018 to 2020/21) with a commercial link to Amazon seem to link items not more available, I tried to purchase at Yi Store in Amazon a couple of  new Dome Guard 1080 serial RFUSY73QK27GMA220408, frmw 12.1.47.2_2023 ...

The bad news: I tried all the options offered for Dome in the many versions of Yi-hack-*-*, but no one worked. The good one: Yi cameras are very resilient since I did not obtain any sort of bricking ... and after each reset they bravely resumed working normally from the cloud. The Sd card (32 GB) is recognized and works quietly by recording the video sequences.

IMHO Yi's guys forced some protections against our noble sport of "hacking for rtsp flux". In any case: any ideas about the chipset used in these cameras? And any hopes for hacking these new versions?

**Comment by CP52** [comment]
BTW ... the backup procedure  https://github.com/roleoroleo/yi-hack-Allwinner-v2/wiki/Dump-your-backup-firmware-(SD-card) works fine, and now I have a backup ...

**Reply by roleoroleo** [reply]
Please, share your backup.
Maybe I can support this new cam.

**Reply by CP52** [reply]
![root](https://user-images.githubusercontent.com/100157826/214015002-606e9d34-46e2-448c-b614-cbb58ab02507.jpg)
[backup.zip](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/10478693/backup.zip)
Hi Roleo, thank you for the answer! enclosed you find the backup and a snapshot of the SD root. If you are resident in EC I could send you a camera by AMZ ...

**Reply by CP52** [reply]
Oh you roleoroleo! Magical key master! Discoverer of new horizons! Praise and honor to you!
The patch seems to work fine. Now I'm starting to try it intensely and I'll let you know. Many many thanks!

**Comment by roleoroleo** [comment]
Your cam is a r35gb.
The problem is the new version of the fw (12 vs 9).
It is still work in progress.

**Reply by CP52** [reply]
Ok Roleo, I will eagerly await any new news ... Thanks!

**Reply by roleoroleo** [reply]
Do you want to try a beta release?

**Reply by CP52** [reply]
Absolutely Yes! in any case I have another as backup ... and the game is worth the candle!

**Reply by roleoroleo** [reply]
Here it is:
[r35gb_0.2.5.tgz](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/10488241/r35gb_0.2.5.tgz)

**Comment by grochib** [comment]
Hey roleoroleo, 

after already having hacked an yi outdoor 1080p IFUS and two yi home IFUS I just recently bought another yi outdoor 1080p only to realize, that they indeed changed the serials (and most likely the fw along). 

yi Outdoor Camera 1080p
production model: YHS.3017
RFUSY59Y5PJEQR220408

I'm really hoping that you will be able to support us once again with a new firmware and I'm looking forward to this as I do not use those cameras not hacked...
In this sense: do you want my backup as well? I really hope I can support you somehow!

With my best regards

**Reply by roleoroleo** [reply]
What fw version is?
12.x.x.x_2023yyyyyyyyyyy

**Reply by grochib** [reply]
I tried to do a backup, didnt work. I tried with 2 different sd cards (32GB, both formated in Windows 10). do you think this could be because of the sd card?

**Reply by grochib** [reply]
I must admit it was me doing a wrong format before. After formating the sd card again with linux it did work. Attached you can find the backup
[backup.zip](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/10483588/backup.zip)
 
In order to learn about the fw I guess I will have to use the yi app?

**Reply by grochib** [reply]
the firmware is: 
9.0.26.07_202104231123

the camera asks me to update to 
9.0.26.08_202203181055

I was totally expecting a firmware like 12. something...

So which yi hack could I use (if there is any) since I could not find this combination in your list...

**Reply by roleoroleo** [reply]
Try h30ga.
It should work.

**Reply by grochib** [reply]
it did work! Thanks a thousand times!

**Comment by CP52** [comment]
Hi roleoroleo,

after few weeks, it seems that the :8080 web site in not more available. The service is saw by Advance port scanner, it is possible to see the streaming by ONVIF or to connect with SSH, but no web configuration ... 
![scanned port](https://user-images.githubusercontent.com/100157826/223187285-1565900f-069f-4bfd-a0be-a6ae0393a8ec.png)

**Reply by CP52** [reply]
![onvif reply](https://user-images.githubusercontent.com/100157826/223187763-b0432acc-e8a5-43c0-96d7-639815eae8d6.png)

**Reply by roleoroleo** [reply]
Maybe something is corrupted in the sd card.
Please, backup your config and reinstall the hack.

**Comment by CP52** [comment]
Hi,

 

SD extracted, formatted ex novo by PC, Yi cam resetted and reconnected to WIFI, SD inserted and again reformatted by YI cam. YIcam OFF. SD extracted.

Used the r35gb_0.2.5.tgz you sent me and copied in the root of SD.

SD in YI cam, and YIcam ON. Rebooted a couple of time.

 

And, YESSS, now we see the IPCAM web site!

 

Thank you roleoroleo, 

 

Cp

 

Ps It could be some AC supply problems, we experienced a couple of hard blackout due to local medium voltage switchboard.

**Reply by hoctrimede** [reply]
Hello !! for information i have a YRS3521 bought recently on Amazon  (sn : RFUS) 
I put the file  r35gb_0.2.5.tgz that you sent to CP52 and it's work fine . 
However i have some functionnalities that doesn't work . For example 

- in the page "ptzpage=ptz" i have a blank page instead of the move pad.
- in the page "snapshot" i have 
![image](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/7113440/aceb09cb-c2be-4076-9cc7-c7e3a64b966e)

- -
- @roleoroleo : do you know if the file r35gb_0.2.5.tgz is the last version ?

Thanks a lot for the job you made (and make !!)

**Reply by roleoroleo** [reply]
Try to clean the cache of your browser.
If snapshot and stream don't work, probably you used the wrong version of the hack.
Last version is 0.2.7

**Reply by hoctrimede** [reply]
Thank you @roleoroleo 
I upgraded with the last version. 
I will try to clean the cache of my browser (chrome).
Have a nice day and thanks again for your work

**Comment by CP52** [comment]
Hi! There is a new version of the roleoroleo firmware  you could upgrade
directly from firmware 

 

Regards

 

C.P.

Da: hoctrimede ***@***.***> 
Inviato: venerdì 23 giugno 2023 14:23
A: roleoroleo/yi-hack-Allwinner-v2 ***@***.***>
Cc: CP52 ***@***.***>; Author ***@***.***>
Oggetto: Re: [roleoroleo/yi-hack-Allwinner-v2] Recent Yi ipcam Dome Guard
1080 serial RFUS... Are new the Yi cams still compatible? (Discussion #565)

 

Hello !! for information i have a YRS3521 bought recently on Amazon (sn :
RFUS)
I put the file r35gb_0.2.5.tgz that you sent to CP52 and it's work fine .
However i have some functionnalities that doesn't work . For example

*	in the page "ptzpage=ptz" i have a blank page instead of the move
pad.
*	in the page "snapshot" i have
 
<https://user-images.githubusercontent.com/7113440/248265794-aceb09cb-c2be-4
076-9cc7-c7e3a64b966e.png> 

*          

*	 

*	@roleoroleo <https://github.com/roleoroleo>  : do you know if the
file r35gb_0.2.5.tgz is the last version ?

Thanks a lot for the job you made (and make !!)


Reply to this email directly, view it on GitHub
<https://github.com/roleoroleo/yi-hack-Allwinner-v2/discussions/565#discussi
oncomment-6261731> , or unsubscribe
<https://github.com/notifications/unsubscribe-auth/AX4ETAQ2JPGNIOLSGYYHQIDXM
WDD3ANCNFSM6AAAAAAUBT7QAI> .
You are receiving this because you authored the thread.
<https://github.com/notifications/beacon/AX4ETAQ3ENMLO6EQSBOI3QLXMWDD3A5CNFS
M6AAAAAAUBT7QAKWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3T
UL5UWJTQAL6F6G.gif> Message ID:
***@***.***
b.com
<mailto:roleoroleo/yi-hack-Allwinner-v2/repo-discussions/565/comments/626173
***@***.***> >

**Reply by hoctrimede** [reply]
Thanks a lot CP52 : i made it with success but still some pb with the mov pan (see the video attached)

https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/7113440/026a9fd9-f880-43a9-a71a-de105437d24d
